frontend/src/components/MoPicker/MoFacetPicker.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <mo-select
     v-model="selected"
-    :label="facetData.user_key"
+    :label="$t('input_fields.'+facetData.user_key)"
```

Comment:
I prefter translations happening outside of the component. This way it's not dependent on having all its labels in a input_fields object